.circleci/config.yml

```diff
@@ -190,6 +195,38 @@ jobs:
           echo "Setting FALA image $ECR_DEPLOY_IMAGE"
       - cla-end-to-end-tests/behave
 
+  playwright:
+    docker:
+      - image: cimg/python:3.12
+    steps:
+      - checkout
+      - setup_remote_docker
+      - aws-login-to-ecr-and-tagging
+
+      - run:
+          name: Build AWS FALA Docker Image and run Playwright container tests
+          command: |
+            # Up AWS FALA image so playwright can test against it.
```

Comment:
can we clarify this comment, not sure what "Up AWS FALA image" means, it may just be nomenclature that I am unfamiliar with. Looks like we are running it as a docker image so can we just say that?

Reply:
Ah I see. It's because we are doing `docker-compose up` on the FALA image. Upping an image is a Docker term, which means to run the image. 

Sure I'll switch it to just say run AWS docker image instead.